src/RequestCompression/RequestMinCompressionSizeBytes/Configuration.php

```diff
@@ -0,0 +1,45 @@
+<?php
+namespace Aws\RequestCompression\RequestMinCompressionSizeBytes;
+
+use Aws;
+use Aws\RequestCompression\Exception\ConfigurationException;
+
+class Configuration implements ConfigurationInterface
```

Comment:
This is a lot of lines of code to get a number and a boolean; could we get the config another way?